src/popup/components/Form/index.tsx

```diff
@@ -1,21 +1,16 @@
 import React from "react";
+import styled from "styled-components";
 import { Form as FormikForm } from "formik";
-import { FormButton } from "popup/basics";
 
 interface FormProps {
   children: JSX.Element;
```

Comment:
Is this what we should use instead of `React.ReactNode`?

Reply:
ah good catch, this should be `React.ReactNode`